roles/mount_discovery_iso_for_pxe/templates/grub.cfg.j2

```diff
@@ -0,0 +1,5 @@
+set timeout=1
+menuentry 'Install Red Hat Enterprise Linux CoreOS' --class fedora --class gnu-linux --class gnu --class os {
+	linuxefi vmlinuz random.trust_cpu=on ignition.firstboot ignition.platform.id=metal 'coreos.live.rootfs_url=http://{{ HTTPD_PXE_HOST }}/{{ pxe_directory }}/rootfs.img' 
```

Comment:
I think we should add the HTTP port from `HTTPD_PORT_FOR_PXE`

Reply:
I will add that after the fix of the HTTP port